setup.cfg

```diff
@@ -44,7 +44,7 @@ install_requires =
     PyNaCl
     requests
     sqlitedict
-    torch>=1.4
+    torch>=1.4.0
```

Comment:
just a question, why not split the requirements file into mandatory and development and then in setup load the mandatory, then you have to keep updated packages only on one place, something in directions #5193

Reply:
Yeah i saw that separate PR but I guess we can discuss that in the other PR.

Reply:
sure :] that is why I made it separate :]